core/modules/mod.rs

```diff
@@ -33,6 +34,13 @@ pub(crate) use map::ModuleMap;
 
 pub type ModuleId = usize;
 pub(crate) type ModuleLoadId = i32;
+
+#[derive(Debug)]
+pub enum ModuleSourceCode {
+  String(FastString),
+  Bytes(Box<[u8]>),
+}
+// TODO(bartlomieju): remove
```

Comment:
I think I still want to remove `ModuleCode`, or replace what `ModuleSourceCode` is right now